src/webview/apps/lightspeed/roleGeneration/main.ts

```diff
@@ -22,13 +68,22 @@ provideVSCodeDesignSystem().register(
   vsCodeTextField(),
 );
 
+export enum RoleGenerationFileType {
```

Comment:
Do these duplicate [these](https://github.com/ansible/vscode-ansible/pull/1685/files#diff-c32837afb0eedd9adc2dcf8d700dbbf80ba238bcf7679ccdde144bfcbbeba963R71-R80)?

Reply:
It's a dup with lines in src/interfaces/lightspeed.ts. I will take a look to see if I can reuse the same file.